sdk/aqueduct/models/response_models.py

```diff
@@ -1,4 +1,5 @@
 import uuid
```

Comment:
I think it's fine to iterate on the dir structure later. Ideally I'd like to follow the pattern between UI code and backend code to have clear file-level mappings between response definitions (like https://github.com/aqueducthq/aqueduct/blob/main/src/golang/lib/response/workflow.go and https://github.com/aqueducthq/aqueduct/blob/main/src/ui/common/src/handlers/responses/workflow.ts ). But we are still iterating on the plan, and we can change the python side altogether once we start to migrate SDK endpoints to v2.